src/cobra/core/reaction.py

```diff
@@ -239,15 +274,15 @@ def lower_bound(self):
 
     @lower_bound.setter
     @resettable
-    def lower_bound(self, value):
+    def lower_bound(self, value: Union[int, float]) -> None:
```

Comment:
Please make the argument type as `float` and also add a docstring with the necessary information.